### PR TITLE
improve verification for SubscribeType and NameType transactions

### PR DESCRIPTION
**Pull request by NoelBright**
### Proposed changes in this pull request
1. improve verification for SubscribeType and NameType transactions.
2. add subscribe command.
3. add name command.

### Type (put an `x` where ever applicable)
- [x] Bug fix: Link to the issue
- [ ] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### Checklist
Please put an `x` against the checkboxes. Write a small comment explaining if its `N/A` (not applicable)

- [ ] Read the [CONTRIBUTION guidelines](https://github.com/nknorg/nkn#contributing).
- [ ] All the tests are passing after the introduction of new changes.
- [ ] Added tests respective to the part of code I have written.
- [ ] Added proper documentation where ever applicable (in code and README.md).
- [ ] Code has been written according to [NKN-Golang-Style-Guide](https://github.com/nknorg/nkn/wiki/NKN-Golang-Style-Guide)

### Extra information
Any extra information related to this pull request.

### CODE REVIEW

File: chain/pool/txpool.go [hunk 1]
```diff
@@ -23,8 +24,9 @@ var (
 	ErrDoubleSpend           = errors.New("IsDoubleSpend check faild")
 	ErrTxnType               = errors.New("invalidate transaction payload type")
 	ErrNonceTooLow           = errors.New("nonce is too low")
-	ErrDuplicateName         = errors.New("Duplicate NameService operation in one block")
-	ErrNoNameRegistered      = errors.New("name already has not be registered")
+	ErrDuplicateName         = errors.New("name already has be registered")
```
Comment (yilunzhang) [inline chain/pool/txpool.go]: => "name has be registered already"

File: chain/pool/txpool.go [hunk 2]
```diff
@@ -23,8 +24,9 @@ var (
...
+	ErrDuplicateRegistrant   = errors.New("Registrant already has registered name")
```
Comment (yilunzhang) [inline chain/pool/txpool.go]: => "Registrant has already registered this name"

File: chain/pool/txpool.go [hunk 3]
```diff
@@ -23,8 +24,9 @@ var (
...
+	ErrNoNameRegistered      = errors.New("no name has not be registered")
```
Comment (yilunzhang) [inline chain/pool/txpool.go]: => "name has not been registered yet"

Comment (NoelBright) [reply chain/pool/txpool.go]: ok

File: chain/pool/txpool.go (hunk 2 above)
Comment (NoelBright) [reply chain/pool/txpool.go]: ok

File: chain/pool/txpool.go (hunk 1 above)
Comment (NoelBright) [reply chain/pool/txpool.go]: ok

**Comment by NoelBright** [comment]
Yes，it passed my testing.

**Pull request merged**